gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidHandler.java

```diff
@@ -61,11 +54,17 @@ public class CseValidHandler {
     private final NetPositionService netPositionService;
     private final LimitingElementService limitingElementService;
     private final Logger businessLogger;
+    private final CseValidRequestValidator cseValidRequestValidator;
 
-    public CseValidHandler(DichotomyRunner dichotomyRunner, EicCodesConfiguration eicCodesConfiguration,
-                           FileImporter fileImporter, FileExporter fileExporter,
-                           MinioAdapter minioAdapter, NetPositionService netPositionService,
-                           LimitingElementService limitingElementService, Logger businessLogger) {
+    public CseValidHandler(DichotomyRunner dichotomyRunner,
+                           EicCodesConfiguration eicCodesConfiguration,
+                           FileImporter fileImporter,
+                           FileExporter fileExporter,
+                           MinioAdapter minioAdapter,
```

Comment:
This `minioAdapter` is not used in this class anymore and can be removed from the class attributes and from the constructor.